src/core/io/src/4C_io_input_spec.hpp

```diff
@@ -94,6 +95,11 @@ namespace Core::IO
      */
     [[nodiscard]] const Internal::InputSpecImpl& impl() const;
 
+    /**
+     * Get the name of this InputSpec. The name is the identifier used in the input file.
+     */
```

Comment:
```suggestion
    /**
     * Get the name of this InputSpec. The name is used as a key in the input file.
     * The name may be an empty string in case the spec does not have a name.
     */
```